packages/block/types.ts

```diff
@@ -0,0 +1,93 @@
+export type { VNode, VElement } from '../million/types';
+export type Props = Record<string, any>;
+
+export class Hole {
+  key: string;
+  constructor(key: string) {
+    this.key = key;
+  }
+}
```

Comment:
Do we really need a class here? This only holds a string key.

Reply:
switched to object!